tests/integration/configuration-example.json

```diff
@@ -16,7 +16,26 @@
     }
   },
   "test_cases": {
-    "valid_pet_ids": ["1", "2", "3"],
-    "invalid_pet_ids": ["999", "-1", "invalid_id"]
+    "valid_developer_ids": ["21", "22", "23"],
+    "non_existant_developer_ids": ["999", "4500", "1250"],
+    "invalid_developer_ids": ["invalid", "-1", "a2"],
+    "developer_names": ["Bethesda", "Nintendo"],
+
+    "scores": ["2", "4"],
```

Comment:
Score as parameters should be a number: https://github.com/osu-mist/game-curator-api/blob/feature/CO-1517-implement-integration-test/openapi.yaml#L454-L455.